docs/API.md

```diff
@@ -62,28 +62,31 @@
   * `extreme` (boolean): Enables extreme mode, yields an additional 60% performance
     (from 250ms down to 100ms per 10000 ops). There are trade-off's should be
     understood before usage. See [Extreme mode explained](extreme.md). Default: `false`.
-  * `level` (string): one of `'fatal'`, `'error'`, `'warn'`, `'info`', `'debug'`,
+  * `level` (string): One of `'fatal'`, `'error'`, `'warn'`, `'info`', `'debug'`,
     `'trace'`; also `'silent'` is supported to disable logging. Any other value
     defines a custom level and requires supplying a level value via `levelVal`.
     Default: 'info'.
-  * `levelVal` (integer): when defining a custom log level via `level`, set to an
+  * `levelVal` (integer): When defining a custom log level via `level`, set to an
     integer value to define the new level. Default: `undefined`.
-  * `messageKey` (string): the string key for the 'message' in the JSON object. Default `msg`.
-  * `prettyPrint` (boolean|object): enables [pino.pretty](#pretty). This is intended for non-production
+  * `messageKey` (string): The string key for the 'message' in the JSON object. Default `msg`.
+  * `objectKey` (string): The root key name to nest stringified objects under. If not changed any
```

Comment:
I wonder if defaulting to `undefined` would make more sense.

Reply:
I have no preference.